types/react-dom/index.d.ts

```diff
@@ -21,29 +21,44 @@ export function findDOMNode(instance: ReactInstance): Element;
 export function render<P extends DOMAttributes<T>, T extends Element>(
     element: DOMElement<P, T>,
     container: Element | null,
-    callback?: (element: T) => any
+    callback?: () => void
 ): T;
-export function render<P>(
-    element: SFCElement<P>,
+
+export function render(
+    element: Array<DOMElement<any, any>>,
```

Comment:
can you link to any documentation of this API? I didn't know that it was possible to pass an array into `ReactDOM.render`.

Reply:
It's not yet documented but the with React Fiber (v16) you can pass an array of elements to the render function.

See this test case: https://codesandbox.io/s/y310mjkp5v

The test case is using a beta release of react/react-dom (16.0.0-beta.5).

Reply:
Just as a code reference:

https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactChildFiber.js#L565

The parameter `newChild` is basically the `element` passed to the render function. This line in `createChild` checks whether the `element` parameter, which has been passed down to this function, is an array. If so it deals with it respectively which indicates that React-Fiber (v16) supports passing an array of elements to the render function (react-dom).

Reply:
Thanks. Can you add type parameters for props, as in the other overloads?

Reply:
Excuse me, but how would that work? Since we don't have something like "variadic parameter types" in typescript. Am I missing something?

Reply:
It could be `Array<DOMElement<DOMAttributes<any>, any>>>` for the lack of better specificity, to mirror the generic overload above that has `DOMAttributes<T>` as the props.

Reply:
I agree. I've commited the change you suggested. Thanks.